QuizApp/Presentation/Home/QuizCell.swift

```diff
@@ -0,0 +1,125 @@
+import UIKit
+
+class QuizCell: UICollectionViewCell {
+
+    static let reuseIdentifier = "QuizCellID"
```

Comment:
Please use `static let reuseIdentifier = "String(describing: QuizCell.self)"`. That way it is connected to class name.